tsl/test/sql/cagg_refresh_policy_incremental.sql

```diff
@@ -296,7 +296,8 @@ SELECT
         start_offset => INTERVAL '15 days',
         end_offset => NULL,
         schedule_interval => INTERVAL '1 h',
-        buckets_per_batch => 5
+        buckets_per_batch => 5,
+        refresh_newest_first => false
     ) AS job_id \gset
```

Comment:
Would be good to test all three cases: with `refresh_newest_first` set to `true`, `false` and not providing it at all to the function. I think it is sufficient to test that the call of `add_continuous_aggregate_policy` does not generate an error.

Reply:
Done!